compiler/onecc-docker/debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
(Q) Do we need this file?

Reply:
I think we don't need that file.

Yesterday I thought it was required for the process of uploading to my ppa.
Today I tried to upload again without the format file. It was successfully done.

Reply:
@jyoungyun 

I thought it was necessary because there was `format` file in the basic template and [ONE/infra/debian/compiler/source/format](https://github.com/Samsung/ONE/blob/master/infra/debian/compiler/source/format). However, when I searched as you commented, I thought it was not a necessary file, so I removed it and went up to the lunch pad after making the package, and I checked that it worked well. I will remove the file and upload it to the next Merge PR. Thank you so much for your comment. :)